CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.15.1 (2020-02-20)
+
+- Updated the karma configuration to correctly proxy files from `~/assets` during `skyux test`/`skyux watch`. [#214](https://github.com/blackbaud/skyux-sdk-builder/pull/214)
```

Comment:
Capitalize the K in Karma. Also, is the slash between test and watch being used in place of and? If yes, I think we should just say "and" instead of using the slash. If that's not the case, disregard.

Reply:
Updated to address feedback.